lib/modules/minifyJs.es6

```diff
@@ -23,12 +23,12 @@ export default function minifyJs(tree, options, terserOptions) {
 
 
 function stripCdata(js) {
-    const leftStrippedJs = js.replace(/\/\/\s*<!\[CDATA\[/, '');
+    const leftStrippedJs = js.replace(/\/\/\s*<!\[CDATA\[/, '').replace(/\/\*\s*<!\[CDATA\[\s*\*\//, '');
     if (leftStrippedJs === js) {
         return js;
     }
 
-    const strippedJs = leftStrippedJs.replace(/\/\/\s*\]\]>/, '');
+    const strippedJs = leftStrippedJs.replace(/\/\/\s*\]\]>/, '').replace(/\/\*\s*\]\]>\s*\*\//, '');
```

Comment:
Both

```js
// <![CDATA[
const x = 2;
// ]]>
```

and

```js
/* <![CDATA[ */
const x = 2;
/* ]]> */
```

are valid and being used. Add support for handling the second case.